inventory

```diff
@@ -43,6 +43,11 @@ ansible_service_broker_etcd_image_prefix=quay.io/coreos/
 ansible_service_broker_etcd_image_tag=latest
 # END ANSIBLE BROKER CONFIG
 
+[nodes:vars]
+# Mark all nodes as infra and compute nodes (oc >= 3.10)
+openshift_node_groups=[{'name': 'node-config-infra-compute', 'labels': ['node-role.kubernetes.io/infra=true', 'node-role.kubernetes.io/compute=true']}]
```

Comment:
Based on error I see in jenkins, I would try to move this `openshift_node_groups` to `[OSEv3:vars]` section.